.github/workflows/build_python_wheel.yaml

```diff
@@ -0,0 +1,66 @@
+# Reusable workflow for the repositories of the Arcalot organization.
+# This workflow builds a Python "wheel" for distributing the package.
+# It requires the PYPI_TOKEN secret.
+# (It is only run when invoked by another workflow.)
+on:
+  workflow_call:
+    secrets:
+      PYPI_TOKEN:
+        required: true
+env:
+  python_version: ${{ vars.ARCALOT_PYTHON_VERSION }}
+jobs:
+  build_python_wheel:
+    name: build python wheel
+    runs-on: ubuntu-latest
+    steps:
+      - name: version format check
+        if: startsWith(github.event.ref, 'refs/tags/')
```

Comment:
It's just because it was already done that way.  (I briefly contemplated switching it and decided that it would not be better.)

FWIW, I have an update for the workflow which experienced the weird bool/string behavior,  in which the variable now produces a boolean value instead of a stringized one, so it has been restored to what we wanted it to be.  However, I don't have that facility here, so if I changed this reference, it would be to another stringized boolean.  😛